action/protocol/execution/evm/evm.go

```
@@ -121,21 +121,11 @@ func newParams(
 		executorAddr = common.BytesToAddress(actionCtx.Caller.Bytes())
 		getBlockHash = helperCtx.GetBlockHash
 
-		vmConfig            vm.Config
-		contractAddrPointer *common.Address
-		getHashFn           vm.GetHashFunc
+		vmConfig  vm.Config
+		getHashFn vm.GetHashFunc
 	)
 
-	if dest := execution.Contract(); dest != action.EmptyAddress {
-		contract, err := address.FromString(execution.Contract())
-		if err != nil {
-			return nil, errors.Wrapf(err, "failed to decode contract address %s", dest)
```

new code panic instead of return error, no problem?